types/web-element.d.ts

```diff
@@ -360,10 +360,13 @@ export type ScopedElementRect = {
   readonly height: number;
 };
 
-export type DragAndDropDestination = {
+export type DragAndDropDestination = 
+{
   readonly xOffset: number;
   readonly yOffset: number;
-};
+}
+|{readonly x: number; readonly y: number;}
+|WebElement
```

Comment:
@ShreySinha02 You're on the right track here but we do not need to include both the coordinates and WebElement here. You can add `| WebElement` directly in the type definition of `dragAndDrop()`. And perhaps change the above type to `DragAndDropDestinationCoordinates`?